app/controllers/omniauth_callbacks_controller.rb

```diff
@@ -35,9 +35,14 @@ def failure
 
   private
 
+  # TODO: [thepracticaldev/oss] test all of this
```

Comment:
this is to remember that although we test most of this in the system test, there's not much testing of the various branhces of this long method